src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -5445,6 +5445,209 @@ static void F2(object? w)
                 Diagnostic(ErrorCode.WRN_MissingNonNullTypesContextForAnnotation, "?").WithLocation(14, 26));
         }
 
+        [Fact, WorkItem(35748, "https://github.com/dotnet/roslyn/issues/35748")]
+        public void Directive_Annotations_Enable()
```

Comment:
>Directive_Annotations_Enable [](start = 20, length = 28)

We need more test coverage on `#nullable enable annotations` (this PR only includes two). For instance, are we emitting [Nullable] as expected?
We can verify that directly (with a PEModule validator in CompileAndVerify) or indirectly (consumer of library sees annotations as expected). It should be possible to look for existing tests for `#nullable enable` and modify them to also verify `#nullable enable annotations`.

Reply:
To be clear, are you asking for more tests in the vein of `NullableReferenceTypesTests.NonNullTypesContext_XX` as well as some tests to verify emit in AttributeTests.Nullable.cs?

Reply:
Correct. It may also be possible to tweak some of the existing tests to give us coverage for both (for example, we can test `#nullable enable OPTIONAL_ANNOTATIONS` and replace `OPTIONAL_ANNOTATIONS` with either blank or `annotations`).

Reply:
Done

Reply:
I still only see two tests in this file that have `#nullable enable annotations`.

---
In reply to: [295020293](https://github.com/dotnet/roslyn/pull/36464#discussion_r295020293) [](ancestors = 295020293)

Reply:
See NonNullTypesContext_01 through NonNullTypesContext_06

---
In reply to: [295036729](https://github.com/dotnet/roslyn/pull/36464#discussion_r295036729) [](ancestors = 295036729,295020293)